bundled-guava/src/main/java/org/apache/iceberg/GuavaClasses.java

```diff
@@ -55,6 +55,7 @@
 
 // inspired in part by
 // https://github.com/apache/avro/blob/release-1.8.2/lang/java/guava/src/main/java/org/apache/avro/GuavaClasses.java
+@SuppressWarnings("ReturnValueIgnored")
```

Comment:
Need to add this else the build is failing, since we bumped the `gradle-baseline-java` plugin.